UPGRADE-1.14.md

```diff
@@ -76,6 +76,115 @@
+    | `Sylius\Bundle\ApiBundle\CommandHandler\Cart\RemoveItemFromCartHandler`                                    | `sylius_api.command_handler.cart.remove_item_from_cart`                              |
+    | `Sylius\Bundle\ApiBundle\CommandHandler\Cart\InformAboutCartRecalculationHandler`                          | `sylius_api.command_handler.cart.inform_about_cart_recalculation`                    |
+    | `Sylius\Bundle\ApiBundle\CommandHandler\Checkout\UpdateCartHandler`                                        | `sylius_api.command_handler.checkout.update_cart`                                    |
+    | `Sylius\Bundle\ApiBundle\CommandHandler\Checkout\ChooseShippingMethodHandler`                              | `sylius_api.command_handler.checkout.choose_shipping_method`                         |
+    | `Sylius\Bundle\ApiBundle\CommandHandler\Checkout\ChoosePaymentMethodHandler`                               | `sylius_api.command_handler.checkout.choose_payment_method`                          |
+    | `Sylius\Bundle\ApiBundle\CommandHandler\Checkout\CompleteOrderHandler`                                     | `sylius_api.command_handler.checkout.complete_order`                                 |
+    | `Sylius\Bundle\ApiBundle\CommandHandler\Checkout\ShipShipmentHandler`                                      | `sylius_api.command_handler.checkout.ship_shipment`                                  |
+    | `Sylius\Bundle\ApiBundle\CommandHandler\Account\ChangePaymentMethodHandler`                                | `sylius_api.command_handler.account.change_payment_method`                           |
+    | `Sylius\Bundle\ApiBundle\CommandHandler\Cart\ChangeItemQuantityInCartHandler`                              | `sylius_api.command_handler.cart.change_item_quantity_in_cart`                       |
+    | `Sylius\Bundle\ApiBundle\CommandHandler\Catalog\AddProductReviewHandler`                                   | `sylius_api.command_handler.catalog.add_product_review`                              |
+    | `Sylius\Bundle\ApiBundle\CommandHandler\Cart\BlameCartHandler`                                             | `sylius_api.command_handler.cart.blame_cart`                                         |
+    | `Sylius\Bundle\ApiBundle\CommandHandler\Account\ChangeShopUserPasswordHandler`                             | `sylius_api.command_handler.account.change_shop_user_password`                       |
+    | `Sylius\Bundle\ApiBundle\CommandHandler\Account\RequestResetPasswordTokenHandler`                          | `sylius_api.command_handler.account.request_reset_password_token`                    |
+    | `Sylius\Bundle\ApiBundle\CommandHandler\Account\ResendVerificationEmailHandler`                            | `sylius_api.command_handler.request_shop_user_verification_email`                    |
```

Comment:
```suggestion
    | `Sylius\Bundle\ApiBundle\CommandHandler\Account\ResendVerificationEmailHandler`                            | `sylius_api.command_handler.request_shop_user_verification_email`                    |
```
Every other service id in this location is prefixed by `account`, except this one, why

Reply:
My mistake